### ISSUE TITLE
Create Servers Page.

### ISSUE THREAD

**Issue by Retroity** [RavengerGaming/wip2#2]
The last major page that isn't done is the server page, which needs to be filled with content relating to the gaming servers that Ravenger Gaming runs.

The page should hopefully be a list of servers with the following:

- Server Name
- Server Description
- Gamemode (?)
- If possible, server status and number of players online
- A button to join the server.

**Comment by Retroity** [comment]
I'm looking into how we could potentially get server status on a static HTML/JS/CSS only website, either by doing everything in client-side Javascript/jQuery and/or using some sort of API.

So far, I found that you can lookup the status of a source engine server using the Steam API. So, for example, with the one server we have, We could make an API call to this URL:

https://api.steampowered.com/ISteamApps/GetServersAtAddress/v1?addr=162.243.58.147:27015&format=json

This URL leads to the Steam API, and looks up very basic information on a server. In this case, it looks up info on the server at 162.243.58.147:27015, which is the current only Ravenger Gaming Game Server. It does not provide info like number of players or map. If we want to do that we'd need to create some sort of status server that can retrieve this information, and display it. This would have to be done in a way where a static site can retrieve this information from the Status Server.

So we have a way of looking up basic server info, however, because I'm not experienced in Javascript or jQuery, I don't know how to call the API, parse the JSON to get the server status, and display it on the page. The implementation of this would need to:

1. Display something along the lines of "Looking up server status..." in Grey before the server status is looked up.
2. When the lookup is complete and the JSON is parsed to get the status, change the "Looking up server status..." message to either be a green "Online" or a red "Offline" message. 
3. Have a timeout set on the lookup, so if the Steam API call failed (such as if the Steam API is down, Steam is blocked on a user's network, etc) a message along the lines of "Could not look up server status" would display. It would also be nice if this message was displayed if the Steam API returned an error. 
4. Be able to be easily implemented so that it can get and display the statuses of multiple game servers.

So basically we need someone to write something either in jQuery or Javascript that can do this. If anyone would like to help, please let me know.
